.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
```

Comment:
how do you send your code coverage to code climate and other CI

Reply:
Add a before_script and after_script. Added!